build/lib/buildkit.sh

```diff
@@ -45,5 +45,15 @@ if [ -f "/buildkit.sh" ]; then
     (exit $s)
 else
     # skip retry when running locally
-    $CMD "$@"
+    log_file=$(mktemp);
```

Comment:
```suggestion
    log_file=$(mktemp);
    trap "rm -f $log_file" EXIT
```
Is it worth cleaning up after ourselves?